uast/uast.go

```diff
@@ -326,3 +326,37 @@ const (
 func (f IncludeFlag) Is(of IncludeFlag) bool {
 	return f&of != 0
 }
+
+type NodePath []*Node
```

Comment:
We have to explain that the first node is supposed to be the root and the last one one is supposed to be the target or destination of the path.  Depending of the name chosen to explain this, maybe we should change the name of the Node method for that name.